pyseir/load_data.py

```diff
@@ -161,8 +163,8 @@ def load_new_case_data_by_fips(
     observed_new_deaths: array(int)
         Array of new deaths observed each day.
     """
-    county_case_timeseries = combined_datasets.get_timeseries_for_fips(
```

Comment:
Might be good to check that there is only one region (fips) in the region_timeseries

Reply:
Done